CHANGELOG.md

```diff
@@ -2,8 +2,14 @@
 
 ## Unreleased - 4.x
 
+### API breaking Changes
```

Comment:
Technically this reverted the breakage, we can fix the final changelog before GA

Reply:
Yeah since we had separated out the changes into `-beta.1` etc. it was hard to know what to do... I didn't want to remove the breakage from the earlier alpha release as that would be misleading. But yeah, we can consolidate it all when we GA. 👍🏻